redhat-cop/infra-ansible
# Renaming 'config-tools' to 'config-packages'

### What does this PR do?
To make the role more generic, the `config-tools` is being renamed to `config-packages`

### How should this be tested?
See `tests` directory for sample.

Create an inventory with desirable packages/RPMs and run with this role. I.e.:

```
list_of_packages_to_install:
  - 'git'
  - 'ansible'
  - 'virt-manager'
```

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
